src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -657,6 +693,16 @@ instruct vaddL_masked(vReg dst_src1, vReg src2, pRegGov pg) %{
   ins_pipe(pipe_slow);
 %}
 
+instruct vaddHF_masked(vReg dst_src1, vReg src2, pRegGov pg) %{
+  predicate(UseSVE > 0);
+  match(Set dst_src1 (AddVHF (Binary dst_src1 src2) pg));
```

Comment:
Do we have such a case in existing jtreg now?

Reply:
Not at the moment. Thanks for pointing this out. I think for this PR, I will remove the predicated instructions support and for now only keep support for non-masked ones. I will add this support when there will be more focus on the masked versions once VectorAPI with Float16Vector is integrated with mainline. Hope this is ok.

Reply:
Sounds good to me. But I'm worried it may crash with bad ad file on AArch64 if the Vector API java and compiler IR part is ready for HF types, while the AArch64 relative masked rules are missing.  Beacause the masked vector IR have been generated, while the codegen is missing on AArch64. We have to add the HF ops to `match_rule_supported_vector_masked` first, and then remove them when adding the masked version rules. WDYT?

Reply:
It's a good idea to disable masked ops in the backend. That should help not generate the masked IR in the inline expanders. I'll update this PR soon. Thanks for your comments.

Reply:
Done. Thanks!